lib/index.ts

```diff
@@ -59,6 +60,7 @@ declare namespace Cypress {
     device?: string;
     customTags?: string;
     diffTollerancePercent?: number;
+    keepScreenshot?: boolean;
```

Comment:
could we add this only to `vrtTrack` command as the others would not utilise this option ?